### ISSUE TITLE
Assign blocks names that are printed in pytest output?

### ISSUE THREAD

**Issue by naten7k** [nschloe/pytest-codeblocks#72]
First off, this package is awesome, thank you for putting it together! As a longtime R knitr/sweave user, this fills a massive hole in python for me (different use case, but conceptually similar). 

I was wondering if there was any way to add block names to each code block, so that when I run pytest, I can easily tell which tests are which? Right now, it's a little tricky to visually parse this: 

```
============================= test session starts ==============================
platform linux -- Python 3.9.6, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- /usr/local/bin/python3
cachedir: .pytest_cache
rootdir: /builds/neuralink/sw/util/python_packages/nlk_utils_n1plot
plugins: codeblocks-0.12.2
collecting ... collected 20 items
README.md::README.md PASSED                                              [  5%]
README.md::README.md PASSED                                              [  5%]
README.md::README.md PASSED                                              [  5%]
README.md::README.md PASSED                                              [  5%]
README.md::README.md PASSED                                              [  5%]
README.md::README.md PASSED                                              [  5%]
README.md::README.md PASSED                                              [  5%]
README.md::README.md PASSED                                              [  5%]
README.md::README.md PASSED                                              [  5%]
```

Also apologies if this isn't an `issue`, happy to move this somewhere else if need be.

Thanks again!

**Comment by nschloe** [comment]
This is fixed in #73. Test names now include line numbers, e.g.,
```
README.md::line 56
```
Will release shortly.

**Comment by naten7k** [comment]
you are awesome, thank you!!!
